### DISCUSSION TITLE
Tuning NIDS Rules in Security Onion

### DISCUSSION THREAD

**Discussion by MrLasVegasSTL** [Security-Onion-Solutions/securityonion#7121] [Unsupported Versions] [answered]
In relation to setting thresholds, documented at the bottom of this page https://docs.securityonion.net/en/2.3/managing-alerts.html.

Can you suppress by source and destination?

For example, I get alerts for ssh activity, I would like to suppress that rule id for [192.168.1.249]() to [192.168.1.0/24](). Not just .249's SSH activity to anything, or anythings ssh activity to the subnet.

Can that be done? If so is it like this?

thresholding:
  sids:
    2001219:
    - suppress:
      gen_id:
      track: by_src
      ip: [192.168.1.249]()
      track: by_dst        
      ip: [192.168.1.0/24]()

**Comment by dougburks** [comment, marked as answer]
I don't believe Suricata supports this today:
https://suricata.readthedocs.io/en/suricata-6.0.0/configuration/global-thresholds.html
https://redmine.openinfosecfoundation.org/issues/1002

One option might be to apply a BPF to Suricata to ignore the ssh activity between those 2 addresses:
https://docs.securityonion.net/en/2.3/bpf.html

**Reply by MrLasVegasSTL** [reply]
Thanks Doug! Exactly what I was looking for, you are the man
